pkg/interceptors/interceptors.go

```diff
@@ -39,5 +74,8 @@ func GetSecretToken(cs kubernetes.Interface, sr *triggersv1.SecretRef, eventList
 		return nil, err
 	}
 
-	return secret.Data[sr.SecretKey], nil
+	secretValue := secret.Data[sr.SecretKey]
+	cache[cacheKey] = secret.Data[sr.SecretKey]
```

Comment:
Shouldn't the intention here be to set the secret value into our request context cache rather than a local cache variable? I'd imagine this would be a function like `func setCache(req *http.Request) {}`